File: backends/spring-java/src/main/java/dev/stackverse/backend/audit/AuditRepository.java
package dev.stackverse.backend.audit;

import java.util.UUID;
import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.data.jpa.repository.JpaSpecificationExecutor;

public interface AuditRepository extends JpaRepository<AuditEntry, UUID>, JpaSpecificationExecutor<AuditEntry> {
}

File: backends/spring-kotlin/src/main/kotlin/dev/stackverse/backend/audit/AuditRepository.kt
package dev.stackverse.backend.audit

import org.springframework.data.jpa.repository.JpaRepository
import org.springframework.data.jpa.repository.JpaSpecificationExecutor
import java.util.UUID

interface AuditRepository :
    JpaRepository<AuditEntry, UUID>,
    JpaSpecificationExecutor<AuditEntry>
